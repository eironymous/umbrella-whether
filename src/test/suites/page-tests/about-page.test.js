import React from "react";
import About from "../../../pages/about/index";
import { mount, configure } from "enzyme";
import { Provider } from "react-redux";
import Adapter from "enzyme-adapter-react-16";
import store from "../../../state/state";

configure({ adapter: new Adapter() });

test('About page loads', () => {
	let wrapper = mount(
		<Provider store={store}>
			<About activeRoute="about" allRoutes={{about: "about"}} />
		</Provider>
	);

	expect(wrapper).toMatchSnapshot();
});

test('About page loads correctly', () => {
	let wrapper = mount(
		<Provider store={store}>
			<About activeRoute="about" allRoutes={{ about: "about" }} />
		</Provider>
	);

	expect(wrapper.find(".header-text").first().text()).toContain("About");
});