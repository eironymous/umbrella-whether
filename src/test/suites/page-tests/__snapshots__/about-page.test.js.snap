// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page loads 1`] = `ReactWrapper {}`;
